renderpyg/tilemap.py

```diff
@@ -216,13 +216,13 @@ def render_tilemap(tilemap, camera=(0,0), scale=1, **kwargs):
 			center = False
 		elif srcrect:
 			srcrect = pg.Rect(srcrect)
-	except:
+	except Exception as e:
 		raise ValueError(
 			'render_tilemap() cannot parse "{}" as src_rect'.format(dstrect))
 
 	center = kwargs.get('center', False)
 	try:
-		if center and center != True:
```

Comment:
center could be a Vector2, Vector3, Tuple, True, or False/None

This change doesn't work as attended